### PR TITLE
Remove books

### PR DESCRIPTION
**Pull request by Veegas**
@nelsammak @MinaNemr @Mernawaseif @KarimElSheikh @Israa-Elwan  Please review

### CODE REVIEW

**Comment by Mernawaseif** [comment]
Remove book sequence  UML is reviewed, nothing to change.
@Veegas

File: backend/test/delete_books.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+var request = require('supertest');
+var should = require('should');
+var setup = require('./setup');
+var Book = require('../models/book.js');
+var User = require('../models/user.js');
+
```
Comment (nelsammak) [inline backend/test/delete_books.js:6]: @Veegas use one var with commas

**Comment by MinaNemr** [comment]
@Veegas 
documentation review done
the code is well documented

**Comment by Israa-Elwan** [comment]
@Veegas , I don't have any comments on the scenario of the story

**Pull request merged**